hack/make/.build-deb/control

```diff
@@ -15,9 +15,9 @@ Recommends: aufs-tools,
             ${apparmor:Recommends}
 Conflicts: docker (<< 1.5~), docker.io, lxc-docker, lxc-docker-virtual-package
 Description: Docker: the open-source application container engine
- Docker is an open source project to pack, ship and run any application as a
- lightweight container
- .
+ Docker is an open source project to build, ship and run any application as a
+ lightweight container.
+
```

Comment:
Sorry, I don't know the reason behind this; I just spotted the other file has a blank line as well;

@tianon, could you shed some light, and check the other file as well before I merge?